src/subscriptions/subscriptionSelector.js

```diff
@@ -0,0 +1,21 @@
+/* flow */
+import { createSelector } from 'reselect';
+
+import type { GlobalState, Narrow } from '../types';
+import { isStreamOrTopicNarrow } from '../utils/narrow';
+
+export const getActiveNarrow = (state: GlobalState): Narrow => state.chat.narrow;
+
+export const getSubscriptions = (state: GlobalState): Stream[] => state.subscriptions;
+
+export const isSubscribed: boolean = createSelector(
```

Comment:
Name it `getIsActiveStreamSubscribed' - `get` to be more in line with what we do use in other places, and `ActiveStream` - because it is not a general utility function, but is specific to the currently active stream.